src/weekly/export.js

```diff
@@ -94,6 +95,11 @@ async function processSubs (downloadStream: ReadStream, deliveryDate: moment, st
       let selectedExporter = exporters.find(exporter => exporter.useForRow(data)) || rowExporter
       selectedExporter.processRow(data)
     })
+    .on('error', function (data) {
+      console.log('Error processing csv:')
+      console.log(data)
+      return false
+    })
```

Comment:
when I run it from the command line, every once in a while it will log an exception (something about trying to read from a stream that is null) but the results seem to be ok. Whether the exception is thrown or not doesn't depend on the input, so I'll have to investigate a little more to see if this extra logging just reveal a preexisting issue or if the combining of the two streams causes it

Reply:
I'm not seeing any exceptions anymore since I changed the stream concatenation library so hopefully that's fixed